blaze/expr/datetime.py

```diff
@@ -119,6 +129,27 @@ def utcfromtimestamp(expr):
     return UTCFromTimestamp(expr)
 
 
+class nanosecond(DateTime): _dtype = datashape.int64
```

Comment:
OK, the last commit addresses the above and adjusts a test to use a changed expectation.